support/lti.js

```diff
@@ -267,3 +267,40 @@ Cypress.Commands.add('ltiLaunchViaTool', options => {
             });
         });
 });
+
+/**
+ * Returns the launch URL of an LTI application
+ *
+ * The command uses the API of `demo-lti1p3`
+ * (https://github.com/oat-sa/demo-lti1p3/blob/master/doc/api.md#ltiresourcelinkrequest-launch-generation-endpoint)
+ *
+ * @example
+ * cy.ltiLaunchViaTool({
+ *   toolUrl: 'http://demo.lti.app/api/launch',
+ *   authToken: 'q4s7fv80n9eq5z5f7fgs',
+ *   registration: 'default',
+ *   ltiResourceId: '0d3d8b41-7af1-4ad1-9fc0-5f9b1db23287'
+ * });
+ *
+ * @param {ltiOptions} options
+ */
+Cypress.Commands.add('getLtiLaunchUrl', options => {
+    const toolUrl = options.toolUrl;
+    const authToken = options.authToken;
+    const registration = options.registration;
+    const claims = prepareLtiClaims(options);
+
+    cy.request({
+        method: 'POST',
+        url: `${toolUrl}/api/platform/messages/ltiResourceLinkRequest/launch`,
+        auth: {
+            bearer: authToken
+        },
+        body: {
+            registration,
+            claims
+        }
+    }).then(response => {
+        return response.body.link;
+    });
```

Comment:
```suggestion
    }).then(response => response.body.link);
```
The result of running the command should return link, I'm wondering how the asynchronous operation, expose the result with return statement `return cy.request`, is this magic happening inside the Cypress, right?

Reply:
indeed, both with and without return statement work. I tried digging into the documentation to see if I can find something relevant which would explain it but I couldn't

Reply:
No worries, I believe it collects all the promises under the `cy.request` call